src/PowerShell/Help/Microsoft.WinGet.Client/Update-WinGetPackage.md

```diff
@@ -0,0 +1,491 @@
+﻿---
+external help file: Microsoft.WinGet.Client.Cmdlets.dll-Help.xml
+Module Name: Microsoft.WinGet.Client
+online version:
+schema: 2.0.0
+---
+
+# Update-WinGetPackage
+
+## SYNOPSIS
+Update a WinGet package.
+
+## SYNTAX
+
+### FoundSet (Default)
+```
+Update-WinGetPackage [-IncludeUnknown] [-Mode <PSPackageInstallMode>] [-Override <String>] [-Custom <String>]
+ [-Location <String>] [-Log <String>] [-Force] [-Header <String>] [-AllowHashMismatch]
+ [-Architecture <PSProcessorArchitecture>] [-InstallerType <PSPackageInstallerType>] [-Locale <String>]
+ [-Scope <PSPackageInstallScope>] [-SkipDependencies] [-Version <String>] [-Id <String>] [-Name <String>]
+ [-Moniker <String>] [-Source <String>] [[-Query] <String[]>] [-MatchOption <PSPackageFieldMatchOption>]
+ [-ProgressAction <ActionPreference>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### GivenSet
+```
+Update-WinGetPackage [-IncludeUnknown] [-Mode <PSPackageInstallMode>] [-Override <String>] [-Custom <String>]
+ [-Location <String>] [-Log <String>] [-Force] [-Header <String>] [-AllowHashMismatch]
+ [-Architecture <PSProcessorArchitecture>] [-InstallerType <PSPackageInstallerType>] [-Locale <String>]
+ [-Scope <PSPackageInstallScope>] [-SkipDependencies] [[-PSCatalogPackage] <PSCatalogPackage>]
+ [-Version <String>] [-ProgressAction <ActionPreference>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Updates the selected package found by searching the installed packages list.
+
+## EXAMPLES
```

Comment:
Given that there's no direct equivalent of `winget upgrade --all`  in `Update-WinGetPackage`, would it make  sense to add an example here using `Get-WinGetPackage | Where-Object {($_.IsUpdateAvailable -eq $true)} | Update-WinGetPackage` for achieving the same result?

Reply:
Totally, useful examples are always welcome in help docs. Sometimes one may specifically look for examples with `Get-Help <cmdlet> -Examples` too.

I'd propose the more modern (PowerShell 3+) syntax of `Get-WinGetPackage | Where-Object IsUpdateAvailable | Update-WinGetPackage` because it's less busy, but other than that no objections.